### ISSUE TITLE
Nexus IQ reporting Medium Policy violation for setuptools (py3-none-any) 56.0.0 (.whl)

### ISSUE THREAD

**Issue by kumachop2** [signalfx/splunk-otel-collector#2572]
We are using Splunk Otel Agent v0.70.0 for AKS and GKE. Nexus IQ scanning alerted medium vulnerability for setuptools (py3-none-any) 56.0.0 (.whl) and patched version is  65.5.1.

Env: Splunk Observability Cloud
Cloud Provider: AKS and GKE
Splunk Otel Agent Reported versions: v0.68.0 and noticed in latest v0.70.0
CVE Details: sonatype-2014-0148 & https://nvd.nist.gov/vuln/detail/CVE-2022-40897  
[pypa/setuptools vulnerable to Regular Expression Denial of Service (ReDoS) · CVE-2022-40897 · GitHub Advisory Database · GitHub](https://github.com/advisories/GHSA-r9hx-vwmv-q579)
Component Occurrences:
setuptools-56.0.0-py3-none-any.whl located at agent/_work/1/s/splunk-otel-collector_0.70.0.tar/785b62bee9098bec077a3e6b06bf39e9bd149e5010aca9aff04b0207fc5ba9ee/layer.tar/usr/lib/splunk-otel-collector/agent-bundle/lib/python3.8/ensurepip/_bundled 

Please do let me know for additional details and advice on availability of patched version of Splunk Otel Agent.

Appreciate your timely help.

**Comment by jeffreyc-splunk** [comment]
Thanks @kumachop2.  The `setuptools-56.0.0-py3-none-any.whl` file is packaged within https://www.python.org/ftp/python/3.8.10/Python-3.8.10.tgz by default, which we include with the smart agent bundle, but `setuptools` itself is upgraded to the latest available version for each release:
```
$ docker run -e SPLUNK_ACCESS_TOKEN=test -e SPLUNK_REALM=test --rm --entrypoint python quay.io/signalfx/splunk-otel-collector:0.68.0 -c 'import setuptools; print(setuptools.__version__)'
65.6.3
```

We can remove the `setuptools-56.0.0-py3-none-any.whl` file in our next release since it is not used.

**Comment by jeffreyc-splunk** [comment]
> @jeffreyc-splunk Thanks much for update. Please do let me know does it covers [CVE-2023-23931](https://github.com/advisories/GHSA-w7pp-m8wf-vj6r) & [GHSA-x4qr-2fvf-3mr5](https://github.com/advisories/GHSA-x4qr-2fvf-3mr5)

@kumachop2 The `crytography` package should be upgraded to 39.0.1 in our next release to cover those vulnerabilities.

**Comment by kumachop2** [comment]
@jeffreyc-splunk Bunch of Thanks for update.

**Comment by kumachop2** [comment]
@jeffreyc-splunk Would you please let me know the upgraded crytography package 39.0.1 which part of helm release.

**Comment by jeffreyc-splunk** [comment]
@kumachop2 cryptography was updated in the [0.71.0 collector release](https://github.com/signalfx/splunk-otel-collector/releases/tag/v0.71.0):
```
$ docker run -it --rm --entrypoint /usr/lib/splunk-otel-collector/agent-bundle/bin/python quay.io/signalfx/splunk-otel-collector:0.71.0 -c 'import cryptography; print(cryptography.__version__)'
39.0.1
```
This image was included in the [0.71.0 helm chart release](https://github.com/signalfx/splunk-otel-collector-chart/releases/tag/splunk-otel-collector-0.71.0).
